components/sections/Header.jsx

```diff
@@ -0,0 +1,28 @@
+import CustomLink from "../ui/CustomLink";
+
+export default function Header() {
+    return (
+        <section className="mt-6.6">
+            {/* header  */}
+            <div className="flex gap-5.5 flex-col text-2.5xl font-normal leading-7 ">
+                {/* header log */}
+                <div className="">
+                    Ape Unit
+                </div>
+                {/* header content */}
+                <div className="">
+                    Unit➇ is a pioneering technology and marketing 
+                    services company that creates end-to-end digital
+                    experiences for brands such as 
+                    <span className="text-primary-300">
+                        {" "}Google, Philips, Audi, Twitch, Patagonia, eBay {" "}
```

Comment:
it would be better if the text are put in an element that represents text better like a paragraph or a heading not floating alone in the div